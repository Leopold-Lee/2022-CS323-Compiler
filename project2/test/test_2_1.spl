int foo()
{
  int a = 10;
  int b = 11 + c;
  6 = a;
  float a = 1.0;
  return c;
}