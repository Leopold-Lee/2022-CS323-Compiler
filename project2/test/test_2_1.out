Error type 1 at Line 4: undefined variable: c
Error type 6 at Line 5: left side in assignment is rvalue
Error type 1 at Line 7: undefined variable: c